### ISSUE TITLE
Openrw crashes after the first cutscene

### ISSUE THREAD

**Issue by gbudny** [rwengine/openrw#133]
Hi

I want to inform you that Openrw crashes after this cutscene:

https://www.youtube.com/watch?v=Mzbl6neJ4uo&list=PL7BCD02E7D8C6D564&index=2

00:57

```
#  gdb ./rwgame GNU gdb (Ubuntu 7.11-0ubuntu1) 7.11 Copyright (C) 2016 Free Software Foundation, Inc. License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html> This is free software: you are free to change and redistribute it. There is NO WARRANTY, to the extent permitted by law.  Type "show copying" and "show warranty" for details. This GDB was configured as "x86_64-linux-gnu". Type "show configuration" for configuration details. For bug reporting instructions, please see: <http://www.gnu.org/software/gdb/bugs/>. Find the GDB manual and other documentation resources online at: <http://www.gnu.org/software/gdb/documentation/>. For help, type "help". Type "apropos word" to search for commands related to "word"...```
Reading symbols from ./rwgame...(no debugging symbols found)...done.
(gdb) run
Starting program: /home/gbudny/Desktop/openrw-build/rwgame/rwgame 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffefa54700 (LWP 1700)]
Warning: The created OpenGL context does not fully meet the settings that were requested
Requested: version = 1.1 ; depth bits = 32 ; stencil bits = 8 ; AA level = 0 ; core = false ; debug = false
Created: version = 3.3 ; depth bits = 24 ; stencil bits = 8 ; AA level = 0 ; core = false ; debug = false
I [Game] Game directory: /home/gbudny/GTAIII
Max UBO Size: 65536
UBO Alignment: 256
Max batch size: 512
I [Renderer] OpenGL Renderer Version: 3.3.0 NVIDIA 340.96 (GLSL 3.30 NVIDIA via Cg compiler) Vendor: NVIDIA Corporation
[New Thread 0x7fffeb6ec700 (LWP 1704)]
[Thread 0x7fffeb6ec700 (LWP 1704) exited]
[New Thread 0x7fffeb6ec700 (LWP 1705)]
[New Thread 0x7fffe6eea700 (LWP 1706)]
I [Data] Loaded 42 zones from /home/gbudny/GTAIII/data/gta3.zon
I [Game] Started
[New Thread 0x7fffe61e9700 (LWP 1749)]
I [World] Loaded cutscene: BET
[Thread 0x7fffe61e9700 (LWP 1749) exited]
[New Thread 0x7fffe59e8700 (LWP 1762)]
I [World] Loaded cutscene: JB
I [World] Creating Vehicle ID 116 (POLICAR)
I [World] Creating Vehicle ID 117 (ENFORCR)
I [World] Creating Vehicle ID 130 (RUMPO)
I [World] Creating Vehicle ID 96 (PATRIOT)
I [World] Creating Vehicle ID 110 (TAXI)
I [World] Creating Vehicle ID 105 (CHEETAH)
I [World] Creating Vehicle ID 92 (STINGER)
I [World] Creating Vehicle ID 105 (CHEETAH)
I [World] Creating Vehicle ID 92 (STINGER)
I [World] Creating Vehicle ID 111 (KURUMA)
I [World] Creating Vehicle ID 110 (TAXI)
I [World] Creating Vehicle ID 111 (KURUMA)
I [World] Creating Vehicle ID 110 (TAXI)
I [World] Creating Vehicle ID 111 (KURUMA)
[Thread 0x7fffe59e8700 (LWP 1762) exited]
I [World] Creating Vehicle ID 111 (KURUMA)
I [World] Creating Vehicle ID 116 (POLICAR)
I [World] Creating Vehicle ID 116 (POLICAR)
[New Thread 0x7fffe51e7700 (LWP 1903)]
I [World] Loaded cutscene: L1_LG

Thread 1 "rwgame" received signal SIGSEGV, Segmentation fault.
0x00007ffff5867de8 in main_arena () from /lib/x86_64-linux-gnu/libc.so.6
(gdb) backtrace
#0  0x00007ffff5867de8 in main_arena () from /lib/x86_64-linux-gnu/libc.so.6
#1  0x000000000047df9b in game_player_in_zone(ScriptArguments const&) ()
#2  0x00000000004a820d in std::_Function_handler<void (ScriptArguments const&), std::function<void (ScriptArguments const&)> conditional_facade<bool>(bool (*)(ScriptArguments const&))::{lambda(ScriptArguments const&)#1}>::_M_invoke(std::_Any_data const&, ScriptArguments const&) ()
#3  0x00000000004a61e4 in ScriptMachine::executeThread(SCMThread&, int) ()
#4  0x00000000004a6607 in ScriptMachine::execute(float) ()
#5  0x00000000004485cf in RWGame::tick(float) ()
#6  0x000000000044bd4d in RWGame::run() ()
#7  0x000000000043e420 in main # ()```
```

**Comment by JayFoxRox** [comment]
Does it only crash when watching the cutscene or also when skipping it?
I usually skip cutscenes and didn't experience the crash.

Also please enclose your logs in 3 back-ticks like:

\`\`\`
log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 
log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 
\`\`\`

which will be displayed as

```
log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 log 1 
log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 log 2 
```

**Comment by gbudny** [comment]
Unfortunately, I can't skip this cutscene, and I made a mistake. I noticed that Openrw crashes when I see this loading screen between cutscenes:

6:33

https://www.youtube.com/watch?v=2I9YxiAaAvE

**Comment by TacoDeBoss** [comment]
Yeah, this mission is crashing for me everywhere. After I first go to luigi's, and after the cutscene. Also when skipping.

One time when it crashed after the cutscene, it had the camera following a Cartel guy for about 2 seconds before it crashed. Didn't have time to snap a screenshot.

**Comment by gbudny** [comment]
I noticed that this bug also exists in this version: sdl-gl33-port.

**Comment by danhedron** [comment]
This is fixed by #141
